lib/util/git/index.ts

```diff
@@ -200,7 +203,7 @@ async function cleanLocalBranches(): Promise<void> {
  * When we initially clone, we clone only the default branch so how no knowledge of other branches existing.
  * By calling this function once the repo's branchPrefix is known, we can fetch all of Renovate's branches in one command.
  */
-export async function setBranchPrefix(branchPrefix: string): Promise<void> {
+async function setBranchPrefix(branchPrefix: string): Promise<void> {
```

Comment:
@rarkins merge this with `setUserConfig` ?

Reply:
Yes, if it's not used elsewhere

Reply:
It's used in `syncGit()`